tests/integration/conftest.py

```diff
@@ -1169,7 +1169,7 @@ def mock_system(request):
     """Update the agent system in the global DB using the `mocked_system` variable defined in the test module and
        restore the initial one after finishing.
     """
-    system = getattr(request.module, 'mocked_system') if hasattr(request.module, 'mocked_system') else 'RHEL8'
+    system = getattr(request.module, 'mocked_system') if hasattr(request.module, 'mocked_system') else 'RHEL9'
```

Comment:
Modifying this setting affects all ITs using the default function, so it would revert the changes:
```suggestion
    system = getattr(request.module, 'mocked_system') if hasattr(request.module, 'mocked_system') else 'RHEL8'
```

Reply:
Done